scss/components/_panel-secondary.scss

```diff
@@ -0,0 +1,59 @@
+.panel-secondary {
+  $panel-control-margin: 10px;
+
+  box-shadow: $box-shadow-block;
+
+  .panel-body {
+    padding: $case-panel-secondary-padding;
+  }
+
+  .panel-footer {
+    padding: $case-panel-secondary-padding-small $case-panel-secondary-padding;
+  }
+
+  .panel-heading {
+    background: $crm-white;
+    line-height: 1;
+    padding: $case-panel-secondary-padding;
+    position: relative;
+
+    &::after {
+      $border-margin-right: $case-panel-secondary-padding + ($case-panel-secondary-padding - $case-panel-secondary-padding-small);
+
+      border-bottom: 1px solid $gray-light;
+      bottom: 0;
+      content: '';
+      display: block;
+      height: 0;
+      left: $case-panel-secondary-padding-small;
+      position: absolute;
+      width: calc(100% - #{$border-margin-right});
+    }
+  }
+
+  // Vertically aligns a control (select, btn-group, link)
+  // inside the heading
+  .panel-heading-control {
+    display: block;
+    margin-left: 0;
+    margin-top: -13px;
+    position: relative;
+    top: 6px;
```

Comment:
why push up and then push down? Why not `margin-top: -7px;` or `top: -7px;` ?

Reply:
yeah that's the tricky bit (that's why i said that maybe `display: table-cell` would have been a better alternative, but I don't have the time to implement it)

The negative `margin-top` value is to avoid the height of the heading element to be altered by the presence of the control element. The positive `top` value is to align the element vertically without affecting the heading height

See here

![alignment](https://user-images.githubusercontent.com/6400898/47365383-be148d80-d6db-11e8-87a4-a4947f6d5296.gif)

Reply:
![hackerman](https://user-images.githubusercontent.com/1642119/47368475-b4753080-d6af-11e8-9f18-760db536fd95.png)